### PR TITLE
Feature/rw timer keyframes

### PR DESCRIPTION
**Pull request by rwilliams659**
# Som Timer Pull Request Form

## Description
- This adds an animation to the timer so that there is a slow color transition as the time ticks down
- This replaces our previous animation where the color changed abruptly at each 1/4 time checkpoint, which had to be scrapped in the process of fixing a different bug with making sure the timer was receiving & displaying the correct time
## Type of change
- [ ] Bug fix (non-breaking change which fixes an issue)
- [X] New feature (non-breaking change which adds functionality)
- [ ] Breaking change (fix or feature that would cause existing functionality to not work as expected)
- [ ] Other, Please explain:
`Your message here`
## Has This Been Tested?
- [ ] No
- [X] Yes
Please describe the tests that you ran to verify your changes. Provide instructions so we can reproduce. Please also list any relevant details for your test configuration:
`npm test`
# Message for reviewer:
`Your message here`
# Checklist:
- [ ] My code has no unused/commented out code
- [X] My code has no console logs
- [X] I have performed a self-review of my own code
- [X] I have commented on my code, particularly in hard-to-understand areas

**Note: A function was previously left commented out in case we play more with editing this functionality later**

### CODE REVIEW

File: src/Components/CountdownTimer/CountdownTimer.module.scss [hunk 1]
```diff
@@ -108,6 +108,20 @@
   stroke: $salmon;
 }
 
+.baseTimerPathAnimated {
+  @extend .baseTimerPathRemaining;
+  animation: countDown 3600s infinite;
```
Comment (jkwest-93) [inline src/Components/CountdownTimer/CountdownTimer.module.scss:113]: I'm loving this solution! Glad we can find ways to make this work and keep it moving. Plus this transition of colors is so nice!!

**Review by jkwest-93 — APPROVED** [review]

File: src/Components/CountdownTimer/CountdownTimer.module.scss (hunk 1 above)
Comment (Abdeboskey) [reply src/Components/CountdownTimer/CountdownTimer.module.scss:113]: Same! Good job time boxing this solution and re-evaluating our priorities. It's like we're learning how to manage our time.

**Review by Abdeboskey — APPROVED** [review]
A couple of nice solutions today team!

**Review by sciencefixion — APPROVED** [review]

**Pull request merged**